src/m365/commands/docs.ts

```diff
@@ -0,0 +1,38 @@
+import commands from './commands';
+import AnonymousCommand from '../base/AnonymousCommand';
+import { Logger } from '../../cli/Logger';
+import type * as open from 'open';
+const packageJSON = require('../../../package.json');
+
+class DocsCommand extends AnonymousCommand {
+  private _open: typeof open | undefined;
+
+  public get name(): string {
+    return commands.DOCS;
+  }
+
+  public get description(): string {
+    return 'Opens CLI for Microsoft 365 docs webpage in the default browser.';
+  }
+
+  public async commandAction(logger: Logger): Promise<void> {
+
+    if (this.debug) {
+      logger.logToStderr(`Opening CLI for Microsoft 365 docs webpage in the default browser. The URL is ${packageJSON.homepage}`);
+    }
```

Comment:
In fact, we shouldn't always open the browser, only when this config setting is set. Therefore I suggest that:

1. We log the URL of the web page
2. Check whether the user has set the right config setting
3. When the config setting is true --> only then open the URL in the browser, else we just finish the command execution.

Example on how to check for the autoOpenLinksInBrowser config setting can be found here:

https://github.com/pnp/cli-microsoft365/blob/2463cfa472fa7636ab9ab19ca75d800e9dd50ef5/src/m365/app/commands/app-open.ts#L66


I would log this highlighted verbose message only when you are going to open the URL in browser.

Reply:
I have made the changes as you suggested, but have removed the verbose logging since we are now logging the same information before opening the webpage.